pkg/signup/signup_service.go

```diff
@@ -97,7 +97,6 @@ func (s *ServiceImpl) CreateUserSignup(username, userID string) (*crtapi.UserSig
 			Namespace: s.Namespace,
 		},
 		Spec: crtapi.UserSignupSpec{
-			UserID:        userID,
```

Comment:
We still need userID. But now it's used as UserSignup.matadata.name.

Reply:
Hm.. is this part of the spec? i don't see it here: https://github.com/codeready-toolchain/api/blob/master/pkg/apis/toolchain/v1alpha1/usersignup_types.go#L19. 

Does this mean we will still be using it to create a usersignup?

Reply:
It's not in Spec. It's resource name - https://kubernetes.io/docs/concepts/overview/working-with-objects/names/
It's in metadata - https://github.com/codeready-toolchain/api/blob/master/pkg/apis/toolchain/v1alpha1/usersignup_types.go#L74